src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -186,14 +250,13 @@ const styles = StyleSheet.create({
     position: 'absolute',
     opacity: opacityHeaderAnimation
   }),
-  image: (opacityHeaderAnimation) => ({
+  image: {
     width: normalize(48),
     height: normalize(48),
     borderRadius: normalize(24),
     backgroundColor: 'lightgrey',
-    marginRight: 8,
-    opacity: opacityHeaderAnimation
-  }),
+    marginRight: 8
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `image` style is updated. Make sure this change aligns with the desired styling.
```diff
-  image: (opacityHeaderAnimation) => ({
+  image: {
     width: normalize(48),
     height: normalize(48),
     borderRadius: normalize(24),
     backgroundColor: 'lightgrey',
-    marginRight: 8,
-    opacity: opacityHeaderAnimation
+    marginRight: 8
+  },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->